src/components/sky-nav/sky-nav.scss

```diff
@@ -465,6 +469,7 @@ $_masthead-height-sm: ms.step(7);
 /**
  * Apply outline to label when actions are focused by keyboard.
  */
+// stylelint-disable-next-line no-duplicate-selectors
```

Comment:
`no-duplicate-selectors` is a good rule, but in this case I wasn't sure if reformatting this into a parent selector would break the mixin, so I opted to ignore it.